### ISSUE TITLE
Auto Upload Working Intermittently

### ISSUE THREAD

**Issue by gabs247** [nextcloud/android#1275]
### Actual behaviour
- Some photos/videos get autouploaded, some seem to get ignored
- Was working until a major update a few months ago, which also removed the feature of separate photo/video folder locations (so I stopped using it for a while). I thought I'd try it again, but it's still doign the same thing and you still can't specify separate photo/video server locations :(

### Expected behaviour
- Every new photo/video should get autouploaded
 
### Steps to reproduce
1. Setup the AutoUpload feature on Camera folder
2. Check AutoUpload
3. Only see some photos/videos in the AutoUpload queue

### Environment data
Android version: 7.1.1

Device model: Sony Xperia Z5

Stock or customized system: Stock

Nextcloud app version: 1,4,3

Nextcloud server version: OwnCloud

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by mario** [comment]
Hi,

will be fixed soon with 1.5. Sorry for the trouble!

Cheers,
Mario

**Comment by gabs247** [comment]
Thanks Mario. When I first noticed the problem, I was told it would be fixed in the next release as well, but it still hasn't, so could this be a different unknown issue?
Also, what about separating the photo/video location, when will that be re-introduced, if so?

**Comment by mario** [comment]
@gabs247 well, I don't know who told you that, but there was no "next" release yet. Yup, it's separated. I can give you a test APK to play with everything if you wish:

https://cloud.nextcloud.com/s/pAbCOxR71TEhGgR

**Comment by ouafnico** [comment]
When the 1.5 will be out ? The stable release seems not to change often

**Comment by mario** [comment]
Beta soon, and then whenever it feels stable :)

**Comment by ouafnico** [comment]
Thanks  Beta from play store or dev version from fdroid ?

**Comment by mario** [comment]
We will have both ;)

**Comment by tobiasKaminsky** [comment]
Actually there will be the beta from google play store and a rc on fdroid.
On fdroid you will see this if you open the regular nextcloud app entry and select "version". Then there will be "version 1.5.0 rc1" and you can manually upgrade to it.

The dev version from fdroid is complete independent and already contains a little more than the upcoming release.

(that was the reason for renaming it, as people can get confused easily if there are too many "betas") ;-)

**Comment by mario** [comment]
Should be fixed in 2.0.
